spec/rails_helper.rb

```diff
@@ -25,6 +25,32 @@
     end
 end
 
+def stub_omniauth
+  OmniAuth.config.test_mode = true
+  OmniAuth.config.mock_auth[:google] = OmniAuth::AuthHash.new (
```

Comment:
@icorson3, can you tell why this might be failing to pass along an email? My test for omniauth login is working locally, but failing in test. The page displays a "Email can't be blank" message instead of "logged in successfully'. I've adapted this hash from the google omniauth gem documentation and tried moving around the email key value pair, but not seen any success. Is there somewhere else I should be looking too?